ubuntu/flobby_installer.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+create_debpkg="${1:-"False"}"
+
+apt-get update
+apt-get install -y apt-utils \
+git \
+cmake \
+build-essential \
+pkg-config \
+libboost-system1.67.0 \
+libboost1.67-dev \
+libboost-filesystem-dev \
+libboost-chrono1.67-dev \
+libboost-regex-dev \
+libboost-thread1.67-dev \
+libjsoncpp-dev libjsoncpp1 \
+libgraphicsmagick++1-dev \
+libcurl4-gnutls-dev \
+libminizip-dev \
+clang-format \
+libxpm-dev \
+libxcb-screensaver0-dev \
+libxss-dev \
+libqt5opengl5-dev \
```

Comment:
Is libqt5opengl5-dev really needed? Why if so?

Reply:
> Is libqt5opengl5-dev really needed? Why if so?

I removed it. It was a dep earlier for some other stuff I suppose. It is not needed anymore.